### DISCUSSION TITLE
Eigenvalues summary

### DISCUSSION THREAD

**Discussion by lfelipecas** [MaxHalford/prince#142] [Q&A] [answered]
Z is a complete disjunctive table pandas dataframe. I used the followin code:

mca = prince.MCA(n_components=8)
mca_Z = mca.fit(Z)
eigenvalues_summary_Z = mca_Z.eigenvalues_summary

I am getting the following error:
AttributeError: 'MCA' object has no attribute 'eigenvalues_summary'

How can I fix it?

**Comment by MaxHalford** [comment]
Hey! What version of Prince are you using? That shouldn't happen with the latest version.

Also, with the latest version, if you have a complete disjunctive table, then you should use `CA`, not `MCA`. `MCA` basically one-hot encodes the data and calls `CA`.

**Comment by lfelipecas** [comment]
It still not working.

![Screenshot (243)](https://user-images.githubusercontent.com/15110507/232354446-902fcd64-ba98-48f3-942d-9a657eb9ff66.png)

**Reply by MaxHalford** [reply, marked as answer]
You have to upgrade the package. The latest version is [0.10.0](https://pypi.org/project/prince/0.10.0/), not 0.7.1.
